content/en/docs/tutorials/hello-minikube.md

```diff
@@ -190,7 +189,7 @@ Kubernetes [*Service*](/docs/concepts/services-networking/service/).
 
 5. Katacoda environment only: Note the 5-digit port number displayed opposite to `8080` in services output. This port number is randomly generated and it can be different for you. Type your number in the port number text box, then click Display Port. Using the example from earlier, you would type `30369`.
 
-    This opens up a browser window that serves your app and shows the app's response.
+   This opens up a browser window that serves your app and shows the app's response.
```

Comment:
I'm unsure of how is this useful. Could you please explain this change?

Reply:
Normally, an ordered list has an indentation of 3 spaces for its description:

```
1. This is first item of an ordered list
   Its description is here, prefixed with 3 spaces.
1. This is second item of an ordered list
   Its description is here, prefixed with 3 spaces.
```
Its output may look like:

1. This is first item of an ordered list
   Its description is here, prefixed with 3 spaces.
1. This is second item of an ordered list
   Its description is here, prefixed with 3 spaces.

Reply:
Well, I will not change any such kind of indentations later coz most of you think it is useless and wastes time.

Reply:
IMO, making sure we are indenting the items (of an ordered list) by 3 spaces is the right thing to do.
Markdown has a special explanation of 4-spaces indentation. It may treat the text as code snippets.
We have encountered many such annoying cases.